diff --color -ur blueman-2.2.3.orig/blueman/main/indicators/AppIndicator.py blueman-2.2.3/blueman/main/indicators/AppIndicator.py
--- blueman-2.2.3.orig/blueman/main/indicators/AppIndicator.py	2021-10-27 23:53:20.000000000 +0200
+++ blueman-2.2.3/blueman/main/indicators/AppIndicator.py	2021-11-24 17:27:22.156528203 +0100
@@ -2,8 +2,12 @@
 
 import gi
 
-gi.require_version('AppIndicator3', '0.1')
-from gi.repository import AppIndicator3
+try:
+    gi.require_version('AyatanaAppIndicator3', '0.1')
+    from gi.repository import AyatanaAppIndicator3 as AppIndicator3
+except ValueError:
+    gi.require_version('AppIndicator3', '0.1')
+    from gi.repository import AppIndicator3
 from blueman.main.indicators.GtkStatusIcon import build_menu
 
 if TYPE_CHECKING:
diff --color -ur blueman-2.2.3.orig/blueman/plugins/applet/AppIndicator.py blueman-2.2.3/blueman/plugins/applet/AppIndicator.py
--- blueman-2.2.3.orig/blueman/plugins/applet/AppIndicator.py	2021-10-27 23:53:20.000000000 +0200
+++ blueman-2.2.3/blueman/plugins/applet/AppIndicator.py	2021-11-24 17:26:51.113366220 +0100
@@ -8,9 +8,12 @@
 from blueman.plugins.applet.StatusIcon import StatusIconImplementationProvider
 
 try:
-    require_version('AppIndicator3', '0.1')
+    require_version('AyatanaAppIndicator3', '0.1')
 except ValueError:
-    raise ImportError("AppIndicator3 not found")
+    try:
+        require_version('AppIndicator3', '0.1')
+    except ValueError:
+        raise ImportError("AppIndicator3 not found")
 
 
 class AppIndicator(AppletPlugin, StatusIconImplementationProvider):
