Index: blueman-1.23/blueman/plugins/applet/PowerManager.py
===================================================================
--- blueman-1.23.orig/blueman/plugins/applet/PowerManager.py	2011-10-08 21:09:16.000000000 +0200
+++ blueman-1.23/blueman/plugins/applet/PowerManager.py	2012-09-27 02:14:10.481892534 +0200
@@ -243,6 +243,7 @@
 		#return pixbuf
 		if not self.GetBluetoothStatus():
 			return ("blueman-tray-disabled", "bluetooth-disabled") 
+		return ("blueman-tray-active", "bluetooth-active")
 		
 	def on_adapter_added(self, path):
 		adapter = Bluez.Adapter(path)
